prime-router/docs/api/waters-delivery.yml

```diff
@@ -32,7 +32,7 @@ paths:
           schema:
             type: string
         - $ref: 'https://raw.githubusercontent.com/CDCgov/prime-reportstream/master/prime-router/docs/api/components/sort_and_filter.yml#/sortDir'
-        - $ref: 'https://raw.githubusercontent.com/CDCgov/prime-reportstream/master/prime-router/docs/api/components/sort_and_filter.yml#/sortColumn'
+        - $ref: 'https://raw.githubusercontent.com/CDCgov/prime-reportstream/master/prime-router/docs/api/components/sort_and_filter.yml#/sortCol'
```

Comment:
/sortcol

Reply:
the parameter is named all lowercase, but the component itself can use casing. i find that useful for distinguishing